### ISSUE TITLE
Heroku toolbelt can't find nodejs in your home directory

### ISSUE THREAD

**Issue by rehno-lindeque** [NixOS/nixpkgs#14477]
## Issue description

The heroku toolbelt appears to look for nodejs in your home directory, but does not find it.

### Steps to reproduce

```
$ nix-shell -p heroku

[nix-shell] $ heroku --version
heroku-toolbelt/3.42.45 (x86_64-linux) ruby/2.3.0
heroku-cli/4.28.3-144406a (amd64-linux) go1.6
=== Installed Plugins
heroku-apps@1.7.3
heroku-cli-addons@0.3.0
heroku-fork@4.1.1
heroku-git@2.4.5
heroku-local@5.0.1
heroku-orgs@1.0.5
heroku-papertrail@1.0.2
heroku-pipelines@1.1.2
heroku-redisgreen
heroku-run@3.0.0
heroku-spaces@2.1.2
heroku-status@2.1.2

[nix-shell] $ heroku logs
ERROR: fork/exec /home/myusername/.heroku/node-v5.9.1-linux-x64/bin/node: no such file or directory

[nix-shell] $  ls -l /home/myusername/.heroku/node-v5.9.1-linux-x64/bin/node
-rwxr-xr-x 1 myusername users 25134356 Apr  2 11:51 /home/myusername/.heroku/node-v5.9.1-linux-x64/bin/node
```

## Technical details

* System: 16.03.503.3f96280 (Emu)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: 16.03.503.3f96280

**Comment by kamilchm** [comment]
Fixed in #13914

**Comment by rehno-lindeque** [comment]
Thanks!
